clients/admin-ui/cypress/e2e/config-wizard.cy.ts

```diff
@@ -82,5 +83,26 @@ describe("Config Wizard", () => {
       cy.getByTestId("generic-msg");
       cy.getByTestId("error-log").contains("Internal Server Error");
     });
+
+    it("Allows stepping back to the previous step during an in-progress scan", () => {
+      cy.intercept(
+        "POST",
+        "/api/v1/generate",
+
+        (req) => {
+          req.continue((res) => {
+            res.setDelay(10000);
```

Comment:
should remove this—don't want it slowing down our tests! mostly just a handy dev tool

Reply:
There needs to be some sort of delay during the fetch so that Cypress can run the checks against the "in progress" scan - but I meant to make the delay much shorter than this, so good check regardless 👀